scripting/get5/eventlogger.sp

```diff
@@ -181,6 +181,14 @@ public void EventLogger_SeriesEnd(MatchTeam winner, int t1score, int t2score) {
   EventLogger_EndEvent("series_end");
 }
 
+public void EventLogger_SeriesCancel(MatchTeam winner, int t1score, int t2score) {
+  EventLogger_StartEvent();
+  AddTeam(params, "team", winner);
```

Comment:
I don't think this field is worth adding if it's always TeamNone.

Reply:
I left it in there for competition but it also makes sense to remove it

Reply:
I had originally kept it as in the debug you kept it in:
```
LogDebug("Calling Get5_OnSeriesResult(winner=%d, team1_series_score=%d, team2_series_score=%d)",
           MatchTeam_TeamNone, g_TeamSeriesScores[MatchTeam_Team1],
           g_TeamSeriesScores[MatchTeam_Team2]);
```

Removed it in next commit based on your feedback